_episodes/02-makefiles.md

```diff
@@ -204,11 +193,15 @@ make: `isles.dat' is up to date.
 ~~~
 {: .output}
 
-Nothing happens because Make attempts to build the first target it
-finds in the Makefile, the
-[default target]({{ page.root }}/reference.html#default-target), which is
-`isles.dat` which is already up-to-date. We need to explicitly tell Make we want
-to build `abyss.dat`:
+This means that even though we have added yet another rule, the new
+target `abyss.dat` has not been build. What Make actually did when we
+run it without any command line arguments is to try to build its
+[default target]({{ page.root }}/reference.html#default-target) which
+is the first target it finds. Since this is `isles.dat`, which is
+already up-to-date it was not recreated.
+
+If we want to build a non-default target we need to specify it
+explicitely as the command line argument:
```

Comment:
How would you rephrase it to make it more clear?

My aim was to make it more explicit what is meant by "nothing happens", because actually some computation is indeed done and the result is that the target is not build.